bftengine/src/bcstatetransfer/SourceSelector.hpp

```diff
@@ -73,22 +78,32 @@ class SourceSelector {
 
   void addPreferredReplica(uint16_t replicaId) { preferredReplicas_.insert(replicaId); }
 
+  uint16_t currentPrimary() { return currentPrimary_; }
```

Comment:
We should write strange code for gtest.
Lets think of a solution to make the gest friend of source selector.
We want to get rid of one liners getters APIs in SS anyway.
i suggest to remove this one.
See the way we are using friendship with BCStateTran class from gtest.